hack/ci-e2e-test.sh

```diff
@@ -95,6 +98,11 @@ export E2E_SSH_PUBKEY="$(cat ~/.ssh/id_rsa.pub)"
 ./test/tools/integration/provision_master.sh
 
 echo "Running e2e tests..."
+if [[ ! -z "${NUTANIX_E2E_PROXY_HOST:-}" ]]; then
```

Comment:
Ok, so this is a bit hacky and might be error prune, however let's go with it for now. The next step would be, introducing connectivity on a higher level. Like running the control plane stack on Nutanix as well.

Reply:
That's a good suggestion, thanks. The alternative would be to move the machine-controller e2e tests to the same model as the KKP E2E tests and run the control plane in kind, eliminating the VM completely. But I agree this should be seen more as a temporary hack than a permanent solution.